examples/console/package.json

```diff
@@ -8,7 +8,7 @@
   "dependencies": {
     "@jupyterlab/codemirror": "^0.5.0",
     "@jupyterlab/console": "^0.5.1",
-    "@jupyterlab/default-theme": "^0.5.1",
+    "@jupyterlab/theming": "^0.5.1",
```

Comment:
In my PR, I entirely removed the theming assets from the individual packages. The idea was to not hard code those assets into other packages but let the theme itself choose to include them or not. That would allow themes to include the default assets, or provide their own set of them.